modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -363,8 +371,9 @@ export default {
     manuallyPublished() {
       this.saveMenu = this.computeSaveMenu();
     },
-    original() {
+    original(newValue) {
```

Comment:
In the providing component we watch `original` and update `this.originalDoc.ref`, which is compatible with reactivity since `ref` already exists as a property (even if initially null).